lib/fs/exfat.inc

```diff
@@ -0,0 +1,665 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; ExFat Filesystem Implementation
+; Implements ExFat filesystem inside a memory-stream
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(import "lib/fs/fs.inc")
+(import "class/obj/lisp.inc")
+
+;;;;;;;;;;;;;;;;;;;;
+; ExFat Constants
+;;;;;;;;;;;;;;;;;;;;
+
+; Sector and cluster sizes
+(defcvar +exfat_sector_size 512)

```

Comment:
defcvar is for use in VP assembler files to define compiler environment vars. See the defcvar macro.

Here defq should be used.